docs/api-reference.md

```diff
@@ -0,0 +1,3 @@
+# API Reference
+
+- [rest-api specification](https://editor.swagger.io/?url=https://raw.githubusercontent.com/rufsam/protocol-rfcs/master/text/0026-rest-api/rest-api.yaml)
```

Comment:
What about the Node event API ?
https://playground.asyncapi.io/?load=https://raw.githubusercontent.com/luca-moser/protocol-rfcs/rfc/node-event-api/text/0033-node-event-api/0033-node-event-api.yml